backend/btrixcloud/basecrawls.py

```diff
@@ -62,8 +67,12 @@ def __init__(
         self.colls = colls
         self.storage_ops = storage_ops
 
-        self.presign_duration_seconds = (
-            int(os.environ.get("PRESIGN_DURATION_MINUTES", 60)) * 60
+        presign_duration_seconds = (
+            int(os.environ.get("PRESIGN_DURATION_MINUTES", PRESIGN_MINUTES_DEFAULT))
+            * 60
+        )
+        self.presign_duration_seconds = min(
```

Comment:
Maybe we should just apply the maximum to minutes, one less value to keep track of, so it'll be a max of 10079 * 60